BraveWallet/Crypto/Stores/KeyringStore.swift

```diff
@@ -154,6 +154,13 @@ public class KeyringStore: ObservableObject {
       self?.updateKeyringInfo()
     }
   }
+  
+  func validate(password: String, completion: @escaping (Bool) -> Void) {
+    if !keyring.isKeyringCreated {
+      return
```

Comment:
I will add it for completeness, but I don't believe it's needed because a user shouldn't be able to get to the view to validate their password if they keyring hasn't been created yet.